desired_depth: 2.5
start_enabled: false
servo_gain: 0.4
update_freq: 30.0
height_target: 0.5
cam_tilt: 0
